lib/ckb/types/transaction.rb

```diff
@@ -114,6 +114,11 @@ def compute_hash
         blake2b.hexdigest
       end
 
+      def size
+        transaction_serializer = CKB::Serializers::TransactionSerializer.new(self)
+        Utils.hex_to_bin(transaction_serializer.serialize).bytesize + 4
```

Comment:
why extra size is 4

Reply:
I don't know.

Reply:
Where did you get this value?

Reply:
4 bytes for the tx offset cost with molecule array (transactions).

Reply:
Rename to `serialized_size_in_block`, `4` is an extra cost when add transaction to block in serialize.